test/integration/asserts.py

```diff
@@ -0,0 +1,98 @@
+"""A library with assertation helpers for unit tests."""
+
+
+def assertEqual(a, b):
+  """Assert that arguments a and b are equal."""
+  assert a == b
+
+
+def assertGreater(a, b):
+  """Assert that a is greather than b."""
+  assert a > b
+
+
+def assertGreaterEqual(a, b):
+  """Assert that a is greather than or equal to b."""
+  assert a >= b
+
+
+def assertLessEqual(a, b):
+  """Assert that a is less than or equal to b."""
+  assert a <= b
+
+
+def assertNotIn(a, b):
+  """Assert that a is not contained in b."""
+  assert a not in b
+
+
+def assertIn(a, b):
+  """Assert that a is contained in b."""
+  assert a in b
+
+
+def assertBetweenInclusive(a, min_value, max_value):
+  """Assert that the passed value is between min_value and max_value, inclusive."""
+  assertGreaterEqual(a, min_value)
+  assertLessEqual(a, max_value)
+
+
+def assertCounterEqual(counters, name, value):
```

Comment:
Do we need assert helpers like this? Assuming we do switch to the standard unittest library, we could simply assert on:

self.assertEquals(counters[name], value)

And if the counter isn't in the dictionary, it would still "correctly" fail.